draft-gpew-priv-ppm.md

```diff
@@ -380,7 +380,11 @@ in the "type" field (within the PPM URN namespace "urn:ietf:params:ppm:error:"):
 | unrecognizedTask        | An endpoint received a message with an unknown task ID. |
 | outdatedConfig          | The message was generated using an outdated configuration. |
 | staleReport             | Report could not be processed because it arrived too late. |
+| reportFromTheFuture     | Report could not be processed because its timestamp is too far in the future. |
 | invalidHmac             | The aggregate message's HMAC was not valid. |
+| invalidBatchInterval    | The batch interval in the collect or aggregate share request is not valid for the task. |
+| insufficientBatchSize   | There are not enough reports in the batch interval. |
+| privacyBudgetExceeded   | The privacy budget has been exceeded for one or more reports included in the batch interval. |
```

Comment:
We don't want to overload the term "privacy budget", as this gets used in the context of DP.